edenia/antelope-tools
# Fix/missing bp images 1046

### Handle Missing BP Images

### What does this PR do?

- Resolve #1046

### Steps to test

1. Run the project locally with eos mainnet network
2. Go to the dashboard page
3. Check the images on the Block Producer Schedule

### Screenshots

#### EOS 

![imagen](https://user-images.githubusercontent.com/66583677/201400526-b7243571-11d5-4845-972d-97b11134ea59.png)

### Telos 

![imagen](https://user-images.githubusercontent.com/66583677/201400574-733c34d3-cf5b-4cc9-9ee0-33803a7c1311.png)

### Libre

![imagen](https://user-images.githubusercontent.com/66583677/201400627-55cba82b-ec34-45ac-80e8-2b180484b74a.png)
